docs/user-management/manage-users.mdx

```diff
@@ -82,3 +82,17 @@ Logto supports [organizations](/organizations/organization-management) and can m
 The user-related [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer) APIs are mounted at `/api/users` except for the user activities, i.e., user logs `/api/logs?userId=:userId`.
 
 You can manage users through the Management API in several use cases. Such as [advanced user search](/user-management/advanced-user-search), [bulk creation accounts](https://openapi.logto.io/operation/operation-createuser), [invitation-only sign-up](/end-user-flows/sign-up-and-sign-in/disable-user-registration), etc.
+
+## FAQs \{#faqs}
+
+<details>
+
+<summary>
+
+### How to restrict access to certain applications for specific users? \{#how-to-restrict-access-to-certain-application-for-specific-users}
+
+</summary>
+
+You can assign different roles and permissions to users. In each application, validate user roles and restrict access as needed. Refer to Authorization: [Role-based access control](/authorization/role-based-access-control).
```

Comment:
I don't think we can limit user access to a certain app before authentication, due to our Omni-sign-in nature.